packages/app-store/_utils/stripe.ts

```diff
@@ -70,5 +70,5 @@ export async function getStripeCustomerId(user: UserType): Promise<string> {
 
 const stripePrivateKey = process.env.STRIPE_PRIVATE_KEY || "";
 export const stripe = new Stripe(stripePrivateKey, {
-  apiVersion: "2020-08-27",
```

Comment:
Are we sure this isn’t going to cause issues? This is big jump

Reply:
It is, but I inspected this and the stripe typing is fairly good, types will fail if our interfaces don't match 🙏